src/main.js

```diff
@@ -1,41 +1,41 @@
-import {createMenuTemplate} from './view/menu.js';
-import {createFilterTemplate} from './view/filter.js';
-import {createProfileTemplate} from './view/profile.js';
-import {createFilmCardTemplate} from './view/film-card.js';
+import SiteMenuView from './view/menu.js';
+import FilmFilterView from './view/filter.js';
+import ProfileDetailsView from './view/profile.js';
+import FilmCardView from './view/film-card.js';
 import {generatefilmCard} from './mock/film-card.js';
-import {createShowMoreButton} from './view/show-more.js';
-import {createFilmsQuantity} from './view/films-quantity.js';
-import {createFilmPopupTemplate} from './view/popup.js';
+import ShowMoreButtonView from './view/show-more.js';
+import FilmQuantityView from './view/films-quantity.js';
+import FilmDetailsView from './view/popup.js';
+import {render, RenderPosition} from './utils.js';
+import BoardView from './view/board.js';
+import CardListView from './view/card-list.js';
+
 
 const CARD_COUNT = 20;
 const CARD_COUNT_PER_STEP = 5;
 
-const render = (container, template, place) => {
-  container.insertAdjacentHTML(place, template);
-};
 
 const cards = new Array(CARD_COUNT).fill().map(generatefilmCard);
-
 const siteMainElement = document.querySelector('.main');
 
-render(siteMainElement, createMenuTemplate(), 'afterbegin');
-render(siteMainElement, createFilterTemplate(), 'afterbegin');
+const renderCard = (container, card) => {
+  const cardComponent = new FilmCardView(card);
+  render(container, cardComponent.getElement(), RenderPosition.BEFOREEND);
```

Comment:
in such case defining extra variable is redundant
```suggestion
  render(container, new FilmCardView(card).getElement(), RenderPosition.BEFOREEND);
```